pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.microsoft.azure</groupId>
     <artifactId>spring-data-cosmosdb</artifactId>
-    <version>2.2.5-SNAPSHOT</version>
+    <version>2.3.0</version>
```

Comment:
Can we change this to release version 2.3.0-SNAPSHOT ?
We cannot release 2.3.0 right now because of dependency on track 1 library (Cosmos SDK v3) . 
We need to take dependency on track 2 library of Cosmos SDK (v4) - which will cause breaking changes. So for now, we can only release milestone or SNAPSHOT builds.

Reply:
hi Kushagra, is it possible to release 2.3.0 based on track 1 library. And targeting track 2 library with new version schema? For the breaking changes, the next major version should be 3.0.0

Reply:
Also, since we are actively supporting both v2.1.x and v2.2.x release trains, shouldn't we create a separate release train for this ?